crates/revm/src/evm_impl.rs

```diff
@@ -415,16 +415,55 @@ impl<'a, GSPEC: Spec, DB: Database, const INSPECT: bool> EVMImpl<'a, GSPEC, DB,
         })
     }
 
+    fn create_end(
+        &mut self,
+        inputs: &CreateInputs,
+        ret: InstructionResult,
+        address: Option<Address>,
+        gas: Gas,
+        out: Bytes,
+    ) -> (InstructionResult, Option<Address>, Gas, Bytes) {
```

Comment:
Can you revert back to how it was, as this function (and others `create_precheck `,`call_..`) just wrap around inspector calls and hides just very small functionality and is called only from one place, it is not worth imo.

Reply:
Not exactly what I asked, will make a change to push it over the line.